foreman_kubevirt.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.5', '< 4.0'
 
-  s.add_development_dependency('theforeman-rubocop', '~> 0.0.6')
+  s.add_development_dependency('theforeman-rubocop', '~> 0.1.0')
```

Comment:
https://github.com/theforeman/foreman_ansible/pull/705 it does when this is merged ;)

And yes, we need rather strict versions due to changes in the underlying rubocop versions -- otherwise you get too much red